service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/vm/AzureVmHelper.java

```diff
@@ -52,8 +54,7 @@ public static StepResult deleteVm(
   public static StepResult deleteNetworkInterface(
       AzureCloudContext azureCloudContext,
       ComputeManager computeManager,
-      String networkInterfaceName)
-      throws InterruptedException {
```

Comment:
handle `InterruptedException` internally to this function because I want to call it from a lambda which doesn't do checked exceptions well